x-pack/plugins/observability/e2e/journeys/exploratory_view.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { journey, step, before } from '@elastic/synthetics';
+import { loginToKibana, waitForLoadingToFinish } from '@kbn/synthetics-plugin/e2e/journeys/utils';
+
+journey('Exploratory view', async ({ page, params }) => {
+  before(async () => {
+    await waitForLoadingToFinish({ page });
+  });
+
+  const baseUrl = `${params.kibanaUrl}/app/observability/exploratory-view/?rangeFrom=now-15m&rangeTo=now#?reportType=kpi-over-time&sr=!((dt:ux,mt:___records___,n:undefined-page-views,rdf:(service.name:!()),time:(from:'2021-01-18T12:20:01.682Z',to:'2021-01-18T12:25:27.484Z')))`;
```

Comment:
Try to compose the url e.g. via a local util method or page object so that it's maintainable. Also see if `undefined` in `...undefined-page-views...` could be omitted for a valid test case.